subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/util/PomCustomizer.java

```diff
@@ -76,8 +76,8 @@ static void customizePom(Node root, ShipkitConfiguration conf,
         }
 
         Node license = root.appendNode("licenses").appendNode("license");
-        license.appendNode("name", "The MIT License");
-        license.appendNode("url", repoLink + "/blob/master/LICENSE");
+        license.appendNode("name", conf.getLicenseInfo().getLicense());
```

Comment:
Can we add defaults, Apache 2.0 and blob/master/LICENSE?

Reply:
Sure, good feedback, thx!

Reply:
Cool. Let me know when you're ready.

MIT is 30y old and lawyers no longer like it. Let's default to Apache 2.0.